product_docs/docs/biganimal/release/using_cluster/pgd_cli_ba.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: PGD CLI on BigAnimal
+navTitle: PGD CLI on BigAnimal
+deepToC: true
+---
+
+When running a distributed high-availability cluster on BigAnimal, you can use the [PGD CLI](../../../pgd/latest/cli/) to manage cluster operations, such as switching over write leaders, performing cluster health checks, and viewing various details about nodes, groups, or other aspects of the cluster.
+
+Using the PGD CLI with BigAnimal requires installing it on a client machine, so [install the CLI](../../../pgd/latest/cli/installing_cli/) if you haven't already.
+
+## Connecting to your BigAnimal cluster
+
+### Discovering your database connection string
+
+To connect to your distributed high-availability BigAnimal cluster via the PGD CLI, you need to [discover the database connection string](../../../pgd/latest/cli/discover_connections/) from your BigAnimal console:
+
+1. Log into the [BigAnimal clusters](https://portal.biganimal.com/clusters) view.
+1. In the filter, set **Cluster Type** to **Distributed High Availability** to show only clusters that work with PGD CLI.
+1. Select your cluster.
+1. In the view of your cluster, select the **Connect** tab.
+1. Copy the read/write URI from the connection info. This is your connection string.
+
+### Using the PGD CLI with your database connection string
+
+To use the PGD CLI with your database connection string, use the `--dsn` flag with your PGD CLI command:
+
+```bash
+pgd show-nodes --dsn "<your_connection_string>"
+```
+
+See the [PGD CLI command reference](../../../pgd/latest/cli/command_ref/) for the full range of PGD CLI commands and their descriptions. 
+
+!!! Important
+Your database connection string does not contain the password for your BigAnimal cluster. So, you should have a [`.pgpass` file](https://www.postgresql.org/docs/current/libpq-pgpass.html) properly configured to allow access to the cluster. Your BigAnimal cluster's connection information page has all the necessary information needed for the file. 
```

Comment:
Pgd does not prompt for interactive passwords - this should be up front when explaining this

Reply:
I think I am following this, I made it more clear that PGD doesn't prompt for interactive passwords there.